src/main/java/duke/tasks/Storage.java

```diff
@@ -0,0 +1,192 @@
+package duke.tasks;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+import duke.additionalparsers.DataParser;
+import duke.ui.Notgpt;
+/**
+ * The {@code Storage} class manages the storage, retrieval, and manipulation of tasks.
+ * <p>
+ * Tasks are stored in a text file, and this class provides methods for creating,
+ * modifying, and deleting tasks. The tasks are loaded from a file on initialization
+ * and saved back to the file whenever they are modified.
+ * </p>
+ */
+public class Storage {
+    private ArrayList<Task> store = new ArrayList<>();
```

Comment:
A better name would be something like `taskList`, so it becomes obvious what the `ArrayList` stores.